src/devices/multiterm.ts

```diff
@@ -0,0 +1,91 @@
+import fz from '../converters/fromZigbee';
+import tz from '../converters/toZigbee';
+import * as exposes from '../lib/exposes';
+import {deviceEndpoints} from '../lib/modernExtend';
+import * as reporting from '../lib/reporting';
+import {Definition, Fz, Tz} from '../lib/types';
+import * as utils from '../lib/utils';
+
+const e = exposes.presets;
+const ea = exposes.access;
+
+const endpoints = {
+    silent_mode: 8,
+    heating_cooling: 9,
+    electric_valve: 10,
+};
+
+const states = {
+    silent_mode: ['Inactive', 'Active'],
```

Comment:
```suggestion
    silent_mode: ['inactive', 'active'],
```

Please make all lower case